packages/fileutils-windows/fileutils-windows.0.6.1/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
```

Comment:
The maintainer of the -windows package is you, not Sylvain. I've been told to put myself

Reply:
Fair point. I'll fix this.